tests/tck/features/match/SeekByEdge.feature

```diff
@@ -874,6 +874,423 @@ Feature: Match seek by edge
+      | "Dejounte Murray"    | "Manu Ginobili"      |
+      | "Dejounte Murray"    | "LaMarcus Aldridge"  |
+      | "Dejounte Murray"    | "Tim Duncan"         |
+      | "Dejounte Murray"    | "Tony Parker"        |
+      | "Dejounte Murray"    | "Manu Ginobili"      |
+      | "Dejounte Murray"    | "Marco Belinelli"    |
+      | "Dejounte Murray"    | "Tim Duncan"         |
+      | "Dejounte Murray"    | "LeBron James"       |
+      | "Dejounte Murray"    | "Tony Parker"        |
+      | "Dejounte Murray"    | "Tim Duncan"         |
+      | "Dejounte Murray"    | "Danny Green"        |
+      | "Dejounte Murray"    | "Paul George"        |
+      | "Dejounte Murray"    | "James Harden"       |
+      | "Dejounte Murray"    | "Russell Westbrook"  |
+      | "Dejounte Murray"    | "Ray Allen"          |
+      | "Dejounte Murray"    | "LeBron James"       |
+      | "Dejounte Murray"    | "Carmelo Anthony"    |
+      | "Dejounte Murray"    | "Dwyane Wade"        |
+      | "Blake Griffin"      | "LeBron James"       |
+      | "Blake Griffin"      | "Carmelo Anthony"    |
+      | "Blake Griffin"      | "Dwyane Wade"        |
+      | "Steve Nash"         | "Steve Nash"         |
+      | "Steve Nash"         | "Dwyane Wade"        |
+      | "Steve Nash"         | "Jason Kidd"         |
+      | "Steve Nash"         | "Vince Carter"       |
+      | "Steve Nash"         | "Dirk Nowitzki"      |
+      | "Steve Nash"         | "Steve Nash"         |
+      | "Steve Nash"         | "Steve Nash"         |
+      | "Jason Kidd"         | "Tracy McGrady"      |
+      | "Jason Kidd"         | "Jason Kidd"         |
+      | "Jason Kidd"         | "Steve Nash"         |
+      | "Jason Kidd"         | "Dwyane Wade"        |
+      | "Jason Kidd"         | "Jason Kidd"         |
+      | "Jason Kidd"         | "Dirk Nowitzki"      |
+      | "Jason Kidd"         | "Jason Kidd"         |
+      | "Jason Kidd"         | "Amar'e Stoudemire"  |
+      | "Jason Kidd"         | "Stephen Curry"      |
+      | "Dirk Nowitzki"      | "Dirk Nowitzki"      |
+      | "Dirk Nowitzki"      | "Jason Kidd"         |
+      | "Dirk Nowitzki"      | "Amar'e Stoudemire"  |
+      | "Dirk Nowitzki"      | "Stephen Curry"      |
+      | "Dirk Nowitzki"      | "LeBron James"       |
+      | "Dirk Nowitzki"      | "Chris Paul"         |
+      | "Dirk Nowitzki"      | "Carmelo Anthony"    |
+      | "Dirk Nowitzki"      | "Vince Carter"       |
+      | "Dirk Nowitzki"      | "Dirk Nowitzki"      |
+      | "Dirk Nowitzki"      | "Steve Nash"         |
+      | "Paul George"        | "Paul George"        |
+      | "Paul George"        | "James Harden"       |
+      | "Grant Hill"         | "Kobe Bryant"        |
+      | "Grant Hill"         | "Grant Hill"         |
+      | "Grant Hill"         | "Rudy Gay"           |
+      | "Shaquile O'Neal"    | "Tony Parker"        |
+      | "Shaquile O'Neal"    | "Manu Ginobili"      |
```

Comment:
Do you compare the expected result with the neo4j

Reply:
It's just the result of neo4j.